docs/pages/access-controls/reference.mdx

```diff
@@ -373,3 +374,188 @@ Here is an explanation of the fields used in the `where` and `filter` conditions
 
 Check out our [predicate language](../reference/predicate-language.mdx#scoping-allowdeny-rules-in-role-resources)
 guide for a more in depth explanation of the language.
+
+## Template variables and functions
+
+Teleport provides several template variables and functions to enable more robust access control:
```

Comment:
Is this supposed to supplement the Role Templates page? https://goteleport.com/docs/access-controls/guides/role-templates/#interpolation-rules I'm a bit worried about duplication that may drift over time

Reply:
I think that the linked section is a good introduction to the concept, and this is a reference to all available variables. I don't really see it as duplication.